apps/web/app/components/post-info-hover-icon.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactNode } from 'react';
+import { Flex, styled } from 'leather-styles/jsx';
+import { BasicHoverCard } from './basic-hover-card';
+import { content } from '~/data/content';
+import { InfoCircleIcon } from '@leather.io/ui';
+import { getPostHref } from '~/utils/post-link';
+import { getLearnMoreLink } from '~/features/page/page';
+import { sanitizeContent } from '~/utils/sanitize-content';
+
+interface PostInfoHoverIconProps {
+  postKey: string;
+  children: ReactNode;
+  iconColor?: 'black' | 'white';
+}
+
+export function PostInfoHoverIcon({ postKey, children, iconColor = 'black' }: PostInfoHoverIconProps) {
+  const post = (content.posts as Record<string, any>)[postKey];
+  if (!post) return <>{children}</>;
+  const iconColorToken = iconColor === 'white' ? 'invert' : 'ink.text-subdued';
```

Comment:
I'm pretty sure this won't work in Panda https://panda-css.com/docs/guides/dynamic-styling

Reply:
It seems to work here?

<img width="294" alt="image" src="https://github.com/user-attachments/assets/60cb303b-9734-465b-85f8-dee271de96bf" />